dev/env.multi-cluster-defaults

```diff
@@ -0,0 +1,14 @@
+export FLEET_E2E_NS=fleet-local
+export FLEET_E2E_NS_DOWNSTREAM=fleet-default
+
+export FLEET_E2E_CLUSTER=k3d-upstream
+export FLEET_E2E_CLUSTER_DOWNSTREAM=k3d-downstream
+
+export GIT_HTTP_USER=fleet-ci
+export GIT_HTTP_PASSWORD=foo
+
+export CI_OCI_USERNAME=fleet-ci
+export CI_OCI_PASSWORD=foo
+export CI_OCI_CERTS_DIR=/home/user/src/fleet/FleetCI-RootCA/
```

Comment:
This line means that on most machines, the defaults will not work; should we mention this in the `README`, encouraging users to set up their own `env.{single,multi}-cluster` files?

Reply:
Nice find! I'll look into it, perhaps it's possible to use a relative path to make it work out-of-the-box. If not, I think, we'll need to update the docs as you suggested.